.github/workflows/test.yaml

```diff
@@ -90,153 +90,111 @@ jobs:
           }')
           echo "MATRIX=$MATRIX" >> $GITHUB_ENV
       - name: Set test matrix with 'downstream' option
         if: env.MATRIX_OPTION == 'downstream'
         run: |
           MATRIX=$(jq -nsc '{
               "os": ["ubuntu-latest"],
-              "python-version": ["3.11"]
+              "environment": ["test-311"]
           }')
           echo "MATRIX=$MATRIX" >> $GITHUB_ENV
 
+  pixi_lock:
+    name: Pixi lock
+    runs-on: ubuntu-latest
+    steps:
+      - uses: holoviz-dev/holoviz_tasks/pixi_lock@pixi
+        with:
+          cache: ${{ github.event.inputs.cache == 'true' || github.event.inputs.cache == '' }}
+
   unit_test_suite:
-    name: Unit tests on Python ${{ matrix.python-version }}, ${{ matrix.os }}
-    needs: [pre_commit, setup]
+    name: unit:${{ matrix.environment }}:${{ matrix.os }}
+    needs: [pre_commit, setup, pixi_lock]
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix: ${{ fromJson(needs.setup.outputs.matrix) }}
     timeout-minutes: 120
-    env:
-      DESC: "Python ${{ matrix.python-version }}, ${{ matrix.os }} unit tests"
-      PYTHON_VERSION: ${{ matrix.python-version }}
     steps:
-      - uses: holoviz-dev/holoviz_tasks/install@v0
+      - uses: holoviz-dev/holoviz_tasks/pixi_install@pixi
         if: needs.setup.outputs.code_change == 'true'
         with:
-          name: unit_test_suite
-          python-version: ${{ matrix.python-version }}
-          channel-priority: flexible
-          channels: pyviz/label/dev,conda-forge,nodefaults
-          envs: "-o flakes -o tests -o examples_tests -o tests_ci"
-          cache: ${{ github.event.inputs.cache || github.event.inputs.cache == '' }}
-          conda-update: true
-        id: install
+          environments: ${{ matrix.environment }}
       - name: Check packages latest version
         if: needs.setup.outputs.code_change == 'true'
         run: |
-          conda activate test-environment
-          python scripts/check_latest_packages.py bokeh panel param datashader
-      - name: Download data
-        if: needs.setup.outputs.code_change == 'true'
-        run: |
-          conda activate test-environment
-          bash scripts/download_data.sh
-      - name: doit test_unit
+          pixi run -e ${{ matrix.environment }} check-latest-packages bokeh panel param datashader
+      - name: Test Unit
         if: needs.setup.outputs.code_change == 'true'
         run: |
-          conda activate test-environment
-          doit test_unit
-      - name: test examples
+          pixi run -e ${{ matrix.environment }} test-unit --cov=./holoviews --cov-report=xml
+      - name: Test Examples
         if: needs.setup.outputs.code_change == 'true'
         run: |
-          conda activate test-environment
-          doit test_examples
-      - name: codecov
+          pixi run -e ${{ matrix.environment }} test-example --cov=./holoviews --cov-report=xml --cov-append
+      - uses: codecov/codecov-action@v4
```

Comment:
This is not strictly related to the Pixi, but I noticed that it was failing. So I updated it to use the official workflow.